package.json

```diff
@@ -55,16 +54,16 @@
     "jest": "^24.1.0",
     "jest-extended": "^0.11.2",
     "json-loader": "0.5.7",
-    "klokantech-noto-sans": "git+https://github.com/QwantResearch/fonts.git#f25a802",
     "mapbox-gl": "^1.6.1",
     "mapbox-gl-js-mock": "https://github.com/QwantResearch/mapbox-gl-js-mock.git#7eaee7b",
     "masq-lib": "git+https://github.com/QwantResearch/masq-lib.git#v0.14.3",
-    "node-sass": "^4.9.2",
+    "node-sass": "^4.13.1",
```

Comment:
Is this related to the font stuff?

Reply:
In order to fix compat with node >= 12, I changed this in a separate commit that the reset of the changes